### DISCUSSION TITLE
Vendors package information

### DISCUSSION THREAD

**Discussion by officialxviid** [tattersoftware/codeigniter4-assets#55] [Ideas]
Sorry, I don't mean to be negative here.

If you find it difficult to find a vendor package, I have a package that collects several different vendors that you can use.

You can use the [`officialxviid/disgester`](https://github.com/officialxviid/disgester) package.

for installation:
```bash
composer require officialxviid/disgester
```
> Note:
> It just makes things easier for you. Do not intend to claim the work of others.

<br>

[![Disgester](https://raw.githubusercontent.com/officialxviid/officialxviid/main/assets/disgester/banner.png)](https://github.com/officialxviid/disgester)

**Comment by MGatner** [comment]
Thanks for sharing!

**Reply by officialxviid** [reply]
Ur welcome sir
